app/views/repair_requests/new.html.erb

```diff
@@ -59,7 +59,7 @@
       <%= form.form_group :description do %>
         <%= form.label :description, "Problem description" %>
         <%= form.error_messages :description %>
-        <%= form.text_area :description, class: 'govuk-textarea govuk-!-margin-bottom-1', maxlength: 250, rows: 4, onkeyup: 'characterCount()' %>
+        <%= form.text_area :description, value: get_alert_codes(@cautionary_contacts), class: 'govuk-textarea govuk-!-margin-bottom-1', maxlength: 250, rows: 4, onkeyup: 'characterCount()' %>
```

Comment:
This should be done on the controller, otherwise it will always overwrite the description if an error occurs and the template is rendered again.

Also, it would be better to have the `get_alert_codes` inline, as it's only used here and code scattering makes it harder to read.

Reply:
We should also have this checked on the feature spec.